examples/java/spring-boot/internet-controller/README.md

```diff
@@ -0,0 +1,30 @@
+# Internet Controller (MetaController) function
+
+This project was created to demonstrate how to use MetaController and Knative Functions. This controller shows how to monitor the internet and trigger some internet production tests only when the application is healthy. 
```

Comment:
`how to monitor the internet` -> please be more specific, what exactly is being  monitored ? Some upstream service etc ?